src/styles/main.scss

```diff
@@ -0,0 +1,63 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  @include flex-center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 10px;
+  gap: $main-gap;
+  width: ($main-size * 7) + ($main-gap * 6);
```

Comment:
The arithmetic operations should be wrapped in parentheses to improve readability. Consider changing to: width: (($main-size * 7) + ($main-gap * 6));